recipe/meta.yaml

```diff
@@ -27,7 +27,18 @@ requirements:
   run:
 
 test:
+  requires:
+    - {{ compiler('c') }}
+    - numpy
+    - python
+  files:
+    - debug_blis.c
   commands:
+    # show CPU features of agent we're running on
+    - python -c "import numpy; numpy._pytesttester._show_numpy_info()"
+    - $CC -I$PREFIX/include -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib -lblis -lm -lpthread debug_blis.c        # [unix]
+    - '%CC% -I%LIBRARY_INC% debug_blis.c blis.lib /link /LIBPATH:%LIBRARY_LIB%'   # [win]
```

Comment:
```suggestion
    - 'clang -I%LIBRARY_INC% debug_blis.c blis.lib /link /LIBPATH:%LIBRARY_LIB%'   # [win]
```

Reply:
You need clang in requires too.

Reply:
I had wanted to use the same setup as numpy, which just uses `{{ compiler('c') }}`. Should I use `{{ compiler('clang') }}` here?

Reply:
No, just `clang` and `compiler('c')` is enough.